examples/notebooks/discrete_choice_example.ipynb

```diff
@@ -99,9 +87,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "affair_mod = logit(\"affair ~ occupation + educ + occupation_husb\" \n",
```

Comment:
This may be an opportunity to trim trailing whitespace.

Related: j will likely want the corresponding examples/python/ files to be edited to reflect any changes in the ipynb files

Reply:
Why aren't those just autogenerated by exporting the ipython?  IMO the .py examples should DIAF unless they don't have a notebook version.

Reply:
I agree, but am not the one who needs convincing.

Reply:
I thought somebody volunteered to sync the py files again with the notebook exports.
Also there are a many more notebooks than py files, so those haven't been exported yet.

**If** there are any doc/examples/ py files that don't have a corresponding notebook, then it is a doc bug, and the notebook should be added unless the py file is a duplicate in terms of documented examples.

(The examples in the source code are a different issue and won't go away for some time.)

Reply:
> (The examples in the source code are a different issue and won't go away for some time.)

unless somebody converts/expands them to notebook examples. (which happened with the original notebook examples)

Reply:
I’ll volunteer, if review/merge can be delegated